### ISSUE TITLE
n_threads is not supported in Open3D 0.9.0

### ISSUE THREAD

**Issue by qihao-plus** [weiyithu/SurroundOcc#69]
http://www.open3d.org/docs/0.9.0/python_api/open3d.geometry.TriangleMesh.html?highlight=create_from_point_cloud_poisson
as title

**Comment by lqzhao** [comment]
Yes, our Open3D's version is 0.15.2, so we can activate n_threads. If you Open3D doesn't support multiple threads, you can delete n_threads.
